llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1847,11 +1847,21 @@ static void removeBranchOnCondTrue(VPlan &Plan) {
   using namespace llvm::VPlanPatternMatch;
```

Comment:
Name of `removeBranchOnCondTrue()` should change, to, e.g., `removeBranchOnConst()`?